apps/plugin-browser/src/pages/popup/popup-contents/action-center/automated/select-scope.tsx

```diff
@@ -0,0 +1,303 @@
+            px: 2,
+            py: 1.5,
+            "@media (prefers-color-scheme: dark)": {
+              background: darkModeInputBackgroundColor,
+              borderColor: darkModeBorderColor,
+            },
+          })}
+        >
+          <AddTypeButton
+            disabled={false}
+            onClick={() => setShowTable(true)}
+            label="SELECT TYPE"
+          />
+          <Typography sx={{ fontSize: 14 }}>
+            You must select at least one type to use auto-inference
+          </Typography>
+        </Stack>
+      </Box>
+    );
+  }
+
+  return (
+    <Box>
+      {!anyTypesSelected && <NoTypesSelectedMessage />}
+      <Table
+        sx={{
+          borderRadius: 1,
+          borderStyle: "solid",
+          borderWidth: 1,
+          ...borderColors,
+          th: {
+            background: "#F2F5FA",
+            color: ({ palette }) => palette.gray[90],
+            fontSize: 13,
+            fontWeight: 600,
+            "@media (prefers-color-scheme: dark)": {
+              background: darkModeInputBackgroundColor,
+              color: darkModeInputColor,
+            },
+          },
+          "th, td": {
+            px: 1.5,
+            py: 1,
+            borderStyle: "solid",
+            borderWidth: 1,
+            borderRadius: 1,
+            ...borderColors,
+            "&:first-child": {
+              width: 180,
+            },
+          },
+        }}
+      >
+        <TableHead>
+          <TableRow>
+            <TableCell>
+              <Typography>Types to find</Typography>
+            </TableCell>
+            <TableCell>
+              <Typography>Look for this type on...</Typography>
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {[...rules, ...(draftRule ? [draftRule] : [])].map(
+            ({ entityTypeId, restrictToDomains }) => {
+              return (
+                <TableRow key={entityTypeId ?? "draft-rule"}>
+                  <TableCell>
+                    <EntityTypeSelector
+                      inputHeight={44}
+                      multiple={false}
+                      setTargetEntityTypeIds={(newTargetIds) => {
+                        const targetEntityTypeId = newTargetIds[0];
+                        updateOrAddRule({
+                          targetEntityTypeId,
+                          previousEntityTypeId: entityTypeId,
+                          restrictToDomains,
+                        });
+
+                        // If this was the draft rule (no entityTypeId), we can reset the draft state now
+                        if (!entityTypeId) {
+                          setDraftRule(null);
+                        }
+                      }}
+                      targetEntityTypeIds={entityTypeId ? [entityTypeId] : []}
+                    />
+                  </TableCell>
+                  <TableCell>
+                    <SelectDomains
```

Comment:
The list is created by user input - you should be able to press enter or select an option from the dropdown when typing (though I can't see the dropdown on your demo so maybe something has gone wrong there)

It doesn't look at existing inferences, although it could